app/models/spree/taxon.rb

```diff
@@ -42,10 +38,8 @@ def to_param
     end
 
     def pretty_name
-      ancestor_chain = ancestors.inject("") do |name, ancestor|
-        name + "#{ancestor.name} -> "
-      end
-      ancestor_chain + name.to_s
+      ancestor_chain = ancestors.map(&:name).join(' -> ')
```

Comment:
It looks like `ancestors` was a method provided by the removed gem. So it can't be used here. Maybe it's enough to just refer to the first parent?